PartsApp/ExcelHelper/OperationsExcelHelper.cs

```diff
@@ -395,5 +397,50 @@ private static string xlRCtoA1(int ARow, int ACol, bool RowAbsolute = false, boo
             S = S + ARow.ToString();
             return S;
         }
+
+        public static string GetValidExcelBookTitle(string title)
```

Comment:
There should be no scenario when the method throws an exception.  We don't want, in any case, the program to fail because of a wrong title.
Whenever the requirements are not met, just return a string of a default format, like 
"{CurrentDateTime}_PartsApp'.
Also, add Debug.Assert(true) in case you return the default string. Later we will replace it with logging to know if and when such an error happened.
so it should be smth like:
`if (condition){
    Debug.Assert(true);
    return "{CurrentDateTime}_PartsApp"
}`

Reply:
done